<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>Login</class>
	<widget class="QMainWindow" name="Login">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>400</width>
				<height>300</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Login</string>
		</property>
		<widget class="QWidget" name="centralWidget">
			<layout class="QVBoxLayout" name="verticalLayout">
				<item>
					<layout class="QFormLayout" name="formLayout">
						<item row="0" column="0">
							<widget class="QLabel" name="label_id">
								<property name="text">
									<string>아이디</string>
								</property>
								<property name="styleSheet">
									<string>font-size: 16px;</string>
								</property>
							</widget>
						</item>
						<item row="0" column="1">
							<widget class="QLineEdit" name="idEdit">
								<property name="styleSheet">
									<string>font-size: 16px;</string>
								</property>
							</widget>
						</item>
						<item row="1" column="0">
							<widget class="QLabel" name="label_pw">
								<property name="text">
									<string>비밀번호</string>
								</property>
								<property name="styleSheet">
									<string>font-size: 16px;</string>
								</property>
							</widget>
						</item>
						<item row="1" column="1">
							<widget class="QLineEdit" name="pwEdit">
								<property name="echoMode">
									<enum>QLineEdit::Password</enum>
								</property>
								<property name="styleSheet">
									<string>font-size: 16px;</string>
								</property>
							</widget>
						</item>
					</layout>
				</item>
				<item>
					<layout class="QHBoxLayout" name="buttonLayout">
						<item>
							<widget class="QPushButton" name="loginBtn">
								<property name="text">
									<string>로그인</string>
								</property>
								<property name="styleSheet">
									<string>font-size: 18px; padding: 8px 16px;</string>
								</property>
							</widget>
						</item>
						<item>
							<widget class="QPushButton" name="exitBtn">
								<property name="text">
									<string>종료</string>
								</property>
								<property name="styleSheet">
									<string>font-size: 16px; padding: 6px 12px;</string>
								</property>
							</widget>
						</item>
					</layout>
				</item>
			</layout>
		</widget>
		<widget class="QStatusBar" name="statusBar"/>
	</widget>
	<layoutdefault spacing="6" margin="11"/>
	<resources/>
	<connections/>
</ui>